packages/ui/src/components/shared/export/JsonFile.tsx

```diff
@@ -9,9 +9,18 @@ import { TransferableObject, Props } from 'types';
 interface JsonFileProps extends Props {
   value: JsonBackup;
   object: TransferableObject;
+  title?: string;
+  topInstruction?: React.ReactNode;
+  bottomInstruction?: React.ReactNode;
 }
 
-export default function JsonFile({ value, object }: JsonFileProps): JSX.Element {
+export default function JsonFile({
+  value,
+  object,
+  topInstruction,
+  bottomInstruction,
+  title = '',
```

Comment:
Let's make this a required prop since we don't have logic to show/hide the `<h3>` tag for showing title for this component, same for the `BackupSecretRecoveryPhrase` above as well.

Remove the default `''` value and the optional mark (?) in the Props definition above.